src/components/KPagination/KPagination.vue

```diff
@@ -1,150 +1,138 @@
+            type="button"
+            @click.prevent="changePage(pageCount)"
           >
-            <a
-              aria-label="Go to the last page"
-              data-testid="last-btn"
-              href="#"
-              @click.prevent="changePage(pageCount)"
-            >{{ pageCount }}</a>
-          </li>
-          <li
-            class="pagination-button square"
-            :class="{ disabled: forwardDisabled }"
-            data-testid="next-btn"
-          >
-            <a
-              aria-label="Go to the next page"
-              href="#"
-              @click.prevent="pageForward"
-            >
-              <KIcon
-                class="pagination-arrow-icon"
-                :class="{ 'is-direction-disabled': forwardDisabled }"
-                color="currentColor"
-                icon="arrowRight"
-                :size="KUI_ICON_SIZE_30"
-                view-box="0 0 16 14"
-              />
-            </a>
-          </li>
-        </ul>
-      </template>
-      <PaginationOffset
-        v-else
-        :next-button-disabled="offsetNextButtonDisabled"
-        :prev-button-disabled="offsetPrevButtonDisabled"
-        @get-next-offset="getNextOffset"
-        @get-prev-offset="getPrevOffset"
-      />
-      <div
-        class="page-size-select"
-      >
-        <KDropdown
-          class="page-size-dropdown"
-          data-testid="page-size-dropdown"
-          :items="pageSizeOptions"
-          :kpop-attributes="kpopAttrs"
-          selection-menu
-          show-caret
-          @change="updatePageSize"
-        >
+            {{ pageCount }}
+          </button>
+        </li>
+        <li :class="{ 'arrow-spacing': !disablePageJump }">
           <KButton
             appearance="tertiary"
-            data-testid="page-size-dropdown-trigger"
+            aria-label="Go to the next page"
+            class="pagination-button arrow"
+            data-testid="next-button"
+            :disabled="forwardDisabled ? true : undefined"
+            size="large"
+            type="button"
+            @click.prevent="pageForward"
           >
-            {{ pageSizeText }}
-            <ChevronDownIcon />
+            <template #icon>
+              <ForwardIcon class="pagination-arrow-icon" />
+            </template>
           </KButton>
-        </KDropdown>
-      </div>
+        </li>
+      </ul>
+    </template>
+    <PaginationOffset
+      v-else
+      :next-button-disabled="offsetNextButtonDisabled"
+      :previous-button-disabled="offsetPreviousButtonDisabled"
+      @get-next-offset="getNextOffset"
+      @get-previous-offset="getPreviousOffset"
+    />
+    <div class="page-size-select">
+      <KDropdown
+        class="page-size-dropdown"
+        data-testid="page-size-dropdown"
+        :items="pageSizeOptions"
+        :kpop-attributes="kpopAttrs"
+        selection-menu
+        show-caret
+        @change="updatePageSize"
+      >
+        <KButton
```

Comment:
Does this button need a `type`?

Reply:
Yes, in case it's rendered inside of a `form` element so it doesn't trigger submit action (learnings from the incident we had related to `KSegmentedControl`)

Reply:
I think it was missing the type attribute which is why I commented?

Reply:
Oh, right, fixed now, thanks